### DISCUSSION TITLE
tailsacle with immich

### DISCUSSION THREAD

**Discussion by aviv926** [immich-app/immich#4241] [Q&A] [answered]
I would appreciate any help, I will try to explain the situation as best as possible.

I have a computer with win 10 pro and I run Docker on it which contains immich
immich's files:

```
.env
docker-compose.yml
hwaccel.yml
photos
```

located on disk `H:`

Now that I've activated immich I want to allow myself to access it remotely in order to back up my photos even when I'm not at home, I found tailscalewhich is free and end-to-end encrypted and doesn't require too much prior knowledge
I have installed tailscale on both my computer and my android phone
I was able to get a ping between them and everything works as it should

I tried to set in the immich application the address I got from tailscale together with a port number of 2283 and with the api I got from the immich web but when I tried to connect it says that it can't access the server

I'm sure there's probably something I'm missing in the definition of .env or docker-compose.yml but I really can't figure out what
I tried to read tailscale and immich's doc but I came out completely confused

I would appreciate any help on this!

**Comment by avinal** [comment]
@aviv926 I might be able to help, can you please paste some screenshot or more info?

**Comment by AngelaDMerkel** [comment]
The container you should be attempting to connect to is the proxy container with a default exposed IP of 5800 and whatever the system's Tailscale IP address is.

**Comment by aviv926** [comment, marked as answer]
@avinal @AngelaDMerkel 

Actually this is an old discussion and I was already able to manage with Tailscale at first I thought the setup was complicated but actually it is very easy
The problem for me was that ESET blocked the entry of the remote port which prevented me from accessing my server

In any case, I am attaching here [guides made on Reddit](https://www.reddit.com/r/immich/comments/16tvqm9/immich_over_tailscale_the_esay_way/) and maybe I will write a guide myself if the developers see fit, they will add it to the official documentation

Thank you for your willingness to help!
